includes/association_existing_object.form.inc

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @file
+ * Handles display/submission for associating an EAD with an existing object.
+ */
+
+/**
+ * Defines a file upload form for uploading an associated file.
+ *
+ * @param array $form
+ *   The drupal form.
+ * @param array $form_state
+ *   The drupal form state.
+ * @param AbstractObject $ead
+ *   The object representing the EAD.
+ * @param string $refid
+ *   ID of the level in the EAD content inventory being modified.
+ *
+ * @return array
+ *   The drupal form definition.
+ */
+function islandora_ead_association_existing_object_form(array $form, array &$form_state, AbstractObject $ead, $refid) {
+  module_load_include('inc', 'islandora_ead', 'includes/utilities');
+  $form_state['ead'] = $ead;
+  $form_state['refid'] = $refid;
+
+  $level_info = islandora_ead_get_level_info($ead, $refid);
+  $form['description'] = array(
+    '#markup' => t('Select the object to link to !level. Leave blank to remove current association.', array(
```

Comment:
Did we run this UI past @desmodude750? Not sure if we have a standard way of doing something like this. Seems like could just add a remove button that triggers a separate submit handler which could be clearer?

Reply:
@desmodude750 unsure about it as well, but @mohirt seemed to think this approach is better than the two buttons. And this current submit handler will delete the old association when you provide a new one anyway, so the behaviour seems consistent when you don't provide a new one.